src/libs/utils.js

```diff
@@ -26,6 +26,11 @@ export function shallowEqualObjects (objA, objB) {
   if (!objA || !objB) {
     return false
   }
+
+  if (typeof objA === 'number' && typeof objB === 'number') {
+    return objA === objB
```

Comment:
Hmm.... 🤔 I'm pretty sure this logic would never be hit due to line 23. if you run `shallowEqualObjects(5, 5)` it already returns `true`.

Reply:
Yep, but this is expected.
It is for case when new form value becomes to different number
so `shallowEqualObjects(5, 7)` must return **false** i suppose, but for now it returns **true** as current comparison working only for strings/boolean and array of string i suppose